config/app_config.yml.SAMPLE

```diff
@@ -135,6 +135,9 @@ default: &defaults
   #host: localhost
   #port: 3000
 
+  # base path for the (session) cookie, to support multiple sessions in a multi_coop_install
+  #cookie_base_path: /
```

Comment:
Wouldn't it be easier config if there were just a single `base_path` which both cookie and mailer could use?

Reply:
i changed it to `script_name`to match the rails configuration name